runtime/doc/options.txt

```diff
@@ -128,7 +128,7 @@ A few special texts:
 		Option was set with command line argument |-c|, +, |-S| or
 		|-q|.
 	Last set from environment variable ~
-		Option was set from an environment variable, $VIMINIT,
+		Option was set from an environment variable, $NVIMINIT,
 		$GVIMINIT or $EXINIT.
```

Comment:
GVIMINIT isn't actually checked, so could you remove it? Excuse me if it's out of the scope of this PR.